packages/manager/modules/netapp/src/dashboard/index/style.scss

```diff
@@ -0,0 +1,6 @@
+@import '@ovh-ux/ui-kit/dist/scss/_tokens.scss';
+
+.validate-buttons{
```

Comment:
I don't think this class is used anymore as you switch to the component ovh-manager-inline-property-editor